themethree.html

```diff
@@ -0,0 +1,85 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Calculator</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=League+Spartan:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link href="css/index.css" rel="stylesheet" />
+  </head>
+  <body class="theme-three">
+    <div class="body-container">
+      <div class="title-container">
+        <h2 class="title">calc</h2>
+        <div class="theme-container">
+          <p class="theme-text">THEME</p>
+          <div class="theme-feature">
+            <form name="myForm">
+              <div class="theme-num">
+                <div class="theme-color-container">
+                  <label class="container num" for="mobile-theme-one"
+                    >1
+                    <input
+                      id="mobile-theme-one"
+                      name="mobile-theme"
+                      type="radio"
+                      value="theme-one"
+                      checked="checked"
```

Comment:
you should have theme 3 checked here

Reply:
****
It seems you review an old changes not the latest one